portfolio/src/main/webapp/gallery.js

```diff
@@ -0,0 +1,90 @@
+let galleryImages = document.querySelectorAll(".gallery-img");
+let getLatestOpenedImg;
+//get current width of window inside browser
+let windowWidth = window.innerWidth;
+
+//check if there are gallery images
```

Comment:
This is valid and it works, but my preference would be to put this in a function and call it.

```
function init() {
  if (galleryImages) {
    ...  
  }
}

init();
```